src/common/components/gear-menu-button.scss

```diff
@@ -2,11 +2,21 @@
 // Licensed under the MIT License.
 @import '../../common/styles/colors.scss';
 
+:global(.ms-Button).gear-menu-button:focus::after {
+    border: 2px $always-white solid;
```

Comment:
You can avoid having to use the separate `.high-contrast-theme` style block at all by instead using a theme-aware color variable here (`$neutral-0` is what you'd want here, I think)

Reply:
Applied this change and it worked; this is now available in my most recent commit